### PR TITLE
src/soc/ibm/*: Add file structure to soc

### PR DESCRIPTION
**Pull request by IgorBagnucki**

### CODE REVIEW

File: src/soc/ibm/power9/Kconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+config CPU_POWER9
```
Comment (miczyg1) [inline src/soc/ibm/power9/Kconfig]:
```suggestion
config SOC_IBM_POWER9
```

Comment (miczyg1) [reply src/soc/ibm/power9/Kconfig]: Here and everywhere else

File: src/soc/ibm/power9/Makefile.inc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## SPDX-License-Identifier: GPL-2.0-only
+
+ifeq ($(CPU_POWER9),y)
+
+bootblock-y += bootblock.c
+postcar-y	+= exit_car.S
```
Comment (miczyg1) [inline src/soc/ibm/power9/Makefile.inc]:
```suggestion
postcar-y += exit_car.S
```
Single space everywhere.

File: src/soc/ibm/power9/timer.c [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <delay.h>
+
+void init_timer(void)
+{
```
Comment (miczyg1) [inline src/soc/ibm/power9/timer.c:6]: Timer is already implemented in the src/arch/ppc, see my commit. If there is something related to init_timer there, then move it here. If not, leave a comment that no need to do anything here and arch timer doesn't need any initialization

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: src/soc/ibm/power9/Makefile.inc (hunk 2 above)
Comment (IgorBagnucki) [reply src/soc/ibm/power9/Makefile.inc]: Fixed it

File: src/soc/ibm/power9/timer.c (hunk 3 above)
Comment (IgorBagnucki) [reply src/soc/ibm/power9/timer.c:6]: Added a comment

File: src/soc/ibm/power9/Kconfig (hunk 1 above)
Comment (IgorBagnucki) [reply src/soc/ibm/power9/Kconfig]: Renamed it

File: src/soc/ibm/power9/Kconfig [hunk 4]
```diff
@@ -0,0 +1,15 @@
+config CPU_IBM_POWER9
+	bool
+	help
+	  This example SoC code along with the example/min86 mainboard
+	  should serve as a minimal example how a buildable x86 SoC code
```
Comment (miczyg1) [inline src/soc/ibm/power9/Kconfig]: It is not x86

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: src/soc/ibm/power9/Kconfig (hunk 4 above)
Comment (IgorBagnucki) [reply src/soc/ibm/power9/Kconfig]: I have changed the help, to describe what this code is about.

File: src/soc/ibm/power9/Kconfig [hunk 5]
```diff
@@ -0,0 +1,9 @@
+config CPU_IBM_POWER9
+	bool
+	help
+	  This SoC is template is the minimal template working on POWER9
```
Comment (miczyg1) [inline src/soc/ibm/power9/Kconfig]:
```suggestion
	  This SoC is the minimal template working on POWER9
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**